added get histroy

9Mad-Max5 opened the pull request:
Added the get history function to package.
Not tested deeply

9Mad-Max5 committed:
added get histroy

glensc commented:
please see the changes you've made. looks like you've deleted some code. commented out some code. not described why. so looks bogus. marking as draft. seems incomplete work.

and describe the change, and why.

also don't create pull request from your default branch. all changes to your main branch will end up in this pr.

9Mad-Max5 committed:
adjusted gitignore and added studios

glensc commented on .gitignore:
```diff
@@ -40,3 +40,5 @@ nosetests.xml
 # Virtual environments
 venv
+*.log
+*.json
```
these go to your local git ignore: `.git/info/exclude`

9Mad-Max5 commented:
> please see the changes you've made. looks like you've deleted some code. commented out some code. not described why. so looks bogus. marking as draft. seems incomplete work.
> 
> and describe the change, and why.
> 
> also don't create pull request from your default branch. all changes to your main branch will end up in this pr.

Thanks for the information, wasn't sure that all other changes will be put in this pull request as well.
My initial idea of this pr was just the changes inside sync.py.
I removed get_watched as it was not functional and replaced it according to the trakt api with get_history.
As history is now the correct term for the watched elements.
As the get_watched was not functional I removed it and commented out the deprecation warning.

In the tv.py I added the studios function which was probably not the correct place, but I didn't assume it would affect this pr.
It retrieves according to trakt api documentation a list of the studios from an tvshow.

Correction of the progress call:
-- return self._progress('collection')
++return self._progress('watched')
In line 344 I corrected the code as the give one not using the progress of an TVshow and more the collection which is in the trakt world a difference.

9Mad-Max5 committed:
roll back gitignore changes

9Mad-Max5 committed:
Bring back the removed parts

9Mad-Max5 committed:
back to normal

9Mad-Max5 committed:
set version to alpha to directly install it

glensc commented:
different changes, different pr please. with appropriate description of changes. also remove change for version.

remove draft after you've completed your changes.

glensc commented on trakt/__version__.py:
```diff
@@ -1 +1 @@
-__version__ = "Unknown"
+__version__ = "alpha"
```
this must not be modified. this is filled automatically by CI on release.
